.github/workflows/pytest_publish.yml

```diff
@@ -1,15 +1,25 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 name: Upload Python Package to PyPI Test
 
 on:
-  pull_request:
-    branches:
-      - dev
-    types: [closed]
-
-# Use the following for when we want to publish to the official PyPI from master
-#   push:
-#     tags:
-#       - "v*" # Will trigger for every tag, 'v*'
+  push:
+    tags:
+      - "v*"
```

Comment:
The if statement on line 26 will kill this action since it checks for a pull_request.merged action. I'm not positive, but I believe the push:tags trigger will never intersect with a closed pull request.